packages/dicomImageLoader/src/externalModules.ts

```diff
@@ -13,13 +12,13 @@ const external = {
   },
   get cornerstone() {
     if (!cornerstone) {
-      if (window && window.cornerstone) {
-        cornerstone = window.cornerstone;
+      if (window && (window as any).cornerstone) {
```

Comment:
Just delete the if condition and put it into the assignment
cornerstone = window && (window as any).cornerstone